https://issues.redhat.com/browse/ACM-29840 (#8605)
open-cluster-management/rhacm-docs: search/create_search_collection.adoc

oc apply -f <your-search-collector-config>.yaml
----

> . List the resources in the allow (`data.AllowedResources`) and deny list (`data.DeniedResources`) sections within the config map.

Reviewer: This should be rewritten so that we don't have the spec in parenthesis, not sure how that passed peer review the first time.

> . List the resources in the allow (`data.AllowedResources`) and deny list (`data.DeniedResources`) sections within the config map.

Author: ```suggestion
. List the resources in the `data.AllowedResources` and the `data.DeniedResources` fields within the config map.
```

> .. Collect `services` and `pods` from all `apiGroups`, while also collecting allowing all resources from the `admission.k8s.io` and `authentication.k8s.io` `apiGroups`.

Reviewer: Here if we are coding these, we need to name what they are. Or don't code.

> .. Collect `services` and `pods` from all `apiGroups`, while also collecting allowing all resources from the `admission.k8s.io` and `authentication.k8s.io` `apiGroups`.

Author: ```suggestion
.. Collect services and pods from all `apiGroups`, while also collecting all resources from the `admission.k8s.io` and `authentication.k8s.io` `apiGroups`.
```

> .. Configure your config map to prevent the central collection of `secrets` from all `apiGroups`, while preventing the collection of `policies`, `iampolicies`, and `certificatepolicies` from the `apiGroup` `admission.k8s.io`.

Reviewer: I will pick up on this in a sec.

Reviewer: We kinda need to do the same with all of these, in that we typically don't just use the spec without naming it but I understand that would mean a lot of unnecessary rewrite. Recommend:

Configure your config map to prevent the central collection of secrets from all `apiGroups` specifications, while preventing the collection of `policies`, `iampolicies`, and `certificatepolicies` from the `apiGroup` `admission.k8s.io` specification.

> .. Configure your config map to prevent the central collection of `secrets` from all `apiGroups`, while preventing the collection of `policies`, `iampolicies`, and `certificatepolicies` from the `apiGroup` `admission.k8s.io`.

Author: ```suggestion
.. Configure your config map to prevent the central collection of secrets from all `apiGroups`, while preventing the collection of `policies`, `iampolicies`, and `certificatepolicies` from the `admission.k8s.io` `apiGroup` .
```

       - iampolicies
       - certificatepolicies
----

> *Note:* If you do not provide a config map, all resources are collected by default. When you specify `AllowedResources`, all other resources are automatically excluded. If you list resources in both `AllowedResources` and `DeniedResources`, it is excluded from the collection.

Reviewer: ```suggestion
*Note:* If you do not provide a config map, all resources are collected by default. When you specify `AllowedResources`, all other resources are automatically excluded. If you list resources in both the `AllowedResources` and `DeniedResources` specifications, it is excluded from the collection.
```

Reviewer: And then what is "it" in "it is excluded" -- that should be replaced with what it is to reduce confusion.

> *Note:* If you do not provide a config map, all resources are collected by default. When you specify `AllowedResources`, all other resources are automatically excluded. If you list resources in both `AllowedResources` and `DeniedResources`, it is excluded from the collection.

Author: ```suggestion
*Note:* If you do not provide a config map, all resources are collected by default. When you specify `AllowedResources`, all other resources are automatically excluded. If you list resources in both the `AllowedResources` and `DeniedResources`, resources are excluded from the collection.
```

. Add the `backup` label to include the `search-collector-config` config map to the cluster backups. Run the following command:

+
[source,bash]
----

> oc label configmap search-collector-config -n <namespace-for-search-collector> backup=true

Reviewer 2: Minor correction to the label:
`oc label configmap search-collector-config -n <namespace-for-search-collector> cluster.open-cluster-management.io/backup=""`
From https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.15/html/business_continuity/business-cont-overview#backed-up-resources

> oc label configmap search-collector-config -n <namespace-for-search-collector> backup=true

Author: ```suggestion
oc label configmap search-collector-config -n <namespace-for-search-collector> cluster.open-cluster-management.io/backup=""
```
